pkg/taskrunner/copy.go

```diff
@@ -0,0 +1,55 @@
+package taskrunner
+
+import (
+	"fmt"
+	"os"
+
+	cp "github.com/otiai10/copy"
+	"github.com/rs/zerolog/log"
+)
+
+type copyTask struct {
+	Source      string `json:"src"`
+	Destination string `json:"dest"`
+}
+
+func (e *copyTask) exec(c *Config) error {
+	l := log.With().
+		Str("task type", "copy").
+		Str("dest", e.Destination).
+		Logger()
+
+	// Convert source to list of files in case a glob is used
+	source, err := expandPath(c.Root, e.Source)
+	if err != nil {
+		return err
+	}
+
+	// Ensure output directory exists
+	if err := createDirIfNotExists(e.Destination, 0755); err != nil {
+		return err
+	}
+
+	for _, s := range source {
+		l.Debug().Str("src", s).Msg("Copying files")
+
+		if err := cp.Copy(s, e.Destination); err != nil {
+			return err
+		}
+
+	}
+
+	return nil
+}
+
+func createDirIfNotExists(dir string, perm os.FileMode) error {
+	if _, err := os.Stat(dir); !os.IsNotExist(err) {
+		return err
```

Comment:
This would ignore other errors. What happens if you cannot stat a file because the path was created by `root` but you're `nobody`?

Might want to ensure just the "not exist" case is handled as "continue" while the rest of the errors are popped up.

Additionally, same as above re: error wrapping